Models/AnimationPointerUVs/glTF/AnimationPointerUVs.gltf

```diff
@@ -0,0 +1,13975 @@
+              }
+            },
+            "path": "pointer"
+          }
+        },
+        {
+          "sampler": 31,
+          "target": {
+            "extensions": {
+              "KHR_animation_pointer": {
+                "pointer": "/materials/29/extensions/KHR_materials_clearcoat/clearcoatRoughnessTexture/extensions/KHR_texture_transform/rotation"
+              }
+            },
+            "path": "pointer"
+          }
+        },
+        {
+          "sampler": 32,
+          "target": {
+            "extensions": {
+              "KHR_animation_pointer": {
+                "pointer": "/materials/28/extensions/KHR_materials_clearcoat/clearcoatRoughnessTexture/extensions/KHR_texture_transform/scale"
+              }
+            },
+            "path": "pointer"
+          }
+        },
+        {
+          "sampler": 33,
+          "target": {
+            "extensions": {
+              "KHR_animation_pointer": {
+                "pointer": "/materials/27/extensions/KHR_materials_clearcoat/clearcoatRoughnessTexture/extensions/KHR_texture_transform/offset"
+              }
+            },
+            "path": "pointer"
+          }
+        },
+        {
+          "sampler": 34,
+          "target": {
+            "extensions": {
+              "KHR_animation_pointer": {
+                "pointer": "/materials/26/extensions/KHR_materials_clearcoat/clearcoatTexture/extensions/KHR_texture_transform/rotation"
+              }
+            },
+            "path": "pointer"
+          }
+        },
+        {
+          "sampler": 35,
+          "target": {
+            "extensions": {
+              "KHR_animation_pointer": {
+                "pointer": "/materials/25/extensions/KHR_materials_clearcoat/clearcoatTexture/extensions/KHR_texture_transform/scale"
+              }
+            },
+            "path": "pointer"
+          }
+        },
+        {
+          "sampler": 36,
+          "target": {
+            "extensions": {
+              "KHR_animation_pointer": {
+                "pointer": "/materials/24/extensions/KHR_materials_clearcoat/clearcoatTexture/extensions/KHR_texture_transform/offset"
+              }
+            },
+            "path": "pointer"
+          }
+        },
+        {
+          "sampler": 37,
+          "target": {
+            "extensions": {
+              "KHR_animation_pointer": {
+                "pointer": "/materials/23/pbrMetallicRoughness/baseColorTexture/extensions/KHR_texture_transform/rotation"
+              }
+            },
+            "path": "pointer"
+          }
+        },
+        {
+          "sampler": 38,
+          "target": {
+            "extensions": {
+              "KHR_animation_pointer": {
+                "pointer": "/materials/23/extensions/KHR_materials_transmission/transmissionTexture/extensions/KHR_texture_transform/rotation"
```

Comment:
There are a few of these `transmissionTexture` pointers that point to properties that don't exist.